dev/examples/ThreeBox.tsx

```diff
@@ -11,15 +29,15 @@ function Box(props) {
     // Hold state for hovered and clicked events
     const [clicked, click] = useState(false)
     const time = useTime()
-    const scale = useTransform(time, (t) => Math.sin(t) * 0.5 + 1)
+    // const scale = useTransform(time, (t) => Math.sin(t) * 0.5 + 1)
 
     // Subscribe this component to the render-loop, rotate the mesh every frame
     // useFrame((state, delta) => (ref.current.rotation.x += 0.01))
     // Return the view, these are regular Threejs elements expressed in JSX
     return (
         <motion.mesh
             {...props}
-            scale={scale}
+            // scale={scale}
```

Comment:
Are these comments intentional?

Reply:
Yeah - it doesn't make sense to mix the `useTime` powered scale with animations, I could remove but it's quite interesting to play with

Reply:
Maybe add a comment why it's commented out, so other people know what to do with it too